### PR TITLE
Board::new_empty() -> default()

### PR DESCRIPTION
**Pull request by wcampbell0x2a**

### CODE REVIEW

File: src/game/tile.rs [hunk 1]
```diff
@@ -192,7 +194,6 @@ impl TileGraphic {
         }
         // two pixels around the corner
         for row_index in &[
-            1,
```
Comment (Brian-Catcow-B) [inline src/game/tile.rs:195]: what? the tiles look weird now cuz of this

Comment (wcampbell0x2a) [reply src/game/tile.rs]: uh mb, fixed.

**Comment by wcampbell0x2a** [comment]
oh wait, don't merge this yet

**Comment by wcampbell0x2a** [comment]
> oh wait, don't merge this yet

ok now it's good

**Pull request merged**